engine/src/main/java/org/terasology/rendering/opengl/LwjglFrameBufferObject.java

```diff
@@ -63,13 +63,6 @@ public LwjglFrameBufferObject(ResourceUrn urn, BaseVector2i size) {
             throw new IllegalStateException("Something went wrong with framebuffer! " + result);
         }
 
-        // clear and fill with full alpha
-        FloatBuffer buffer = BufferUtils.createFloatBuffer(16);
-        GL11.glGetFloat(GL11.GL_COLOR_CLEAR_VALUE, buffer);
-        GL11.glClearColor(0f, 0f, 0f, 1f);
-        GL11.glClear(GL11.GL_COLOR_BUFFER_BIT);
-        GL11.glClearColor(buffer.get(), buffer.get(), buffer.get(), buffer.get());  // reset
-
         GL30.glBindFramebuffer(GL30.GL_FRAMEBUFFER, 0);
```

Comment:
We shouldn't be using OpenGL 3.0 directives if we only check for 2.1 support on initialization. What is achieved with this line (binding the display) can be achieved with:

import static org.lwjgl.opengl.EXTFramebufferObject.GL_FRAMEBUFFER_EXT;
import static org.lwjgl.opengl.EXTFramebufferObject.glBindFramebufferEXT;

glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, 0);

which is more in line with the capability checks we do on initialization.

Having said that, of course I can see that's not code you introduced @msteiger.